js/SearchResult.js

```diff
@@ -0,0 +1,77 @@
+class SearchResult {
+  constructor(searchResultElement) {
+    this.searchResultElement = searchResultElement;
+  }
+
+  async getCompanyData(symbol) {
+    try {
+      const url =
+        "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/" +
```

Comment:
Instead of rewriting the API URL ("https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/) you should use (declare) baseUrl variable. Remember the DRY principle.